packages/jest-resolve/src/node_modules_paths.js

```diff
@@ -0,0 +1,47 @@
+import type {Path} from 'types/Config';
```

Comment:
Can you add the copyright header just like in other files?
Also, since it's adapted from `resolve`, an extra annotation is necessary:
```
 /**
  * Copyright (c) 2014, Facebook, Inc. All rights reserved.
  *
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * Adapted from: https://github.com/substack/node-resolve
  * 
  * @flow
  */
```